pkg/policy/mapstate.go

```diff
@@ -143,6 +224,55 @@ func (k Key) PortProtoIsEqual(c Key) bool {
 	return k.DestPort == c.DestPort && k.Nexthdr == c.Nexthdr
 }
 
+// Prefix returns the prefix lenth of the key
+// for indexing it.
+func (k Key) Prefix() uint {
```

Comment:
nit for clarity; a prefix and a prefix length are quite different:
```suggestion
// PrefixLength returns the prefix length of the key
// for indexing it.
func (k Key) PrefixLength() uint {
```